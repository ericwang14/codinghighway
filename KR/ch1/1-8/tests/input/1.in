This is just an example.
There is nothing very special about it. 
