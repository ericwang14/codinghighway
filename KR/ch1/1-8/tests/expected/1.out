Blanks: 11
Tabs: 0
Newlines: 2
